### DISCUSSION TITLE
47419A79-BE29-4B54-AE02-A31AC5D354F5

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#85] [General]
# Exhibit-2

Exhibit-2, aka EXOI-377, is an eclipsing binary system of two K-type main sequence stars located about 146 lightyears from the solar system. Discovered by Exhibit South shortly after the facility opened it has 6 gas giants that orbit the binary. most of these planets were found via RV surveys of the system first before they made any transits themselves. Currently, Exhibit-2 has the most amount of planets that orbit two stars.

https://celestia.mobi/addon?item=47419A79-BE29-4B54-AE02-A31AC5D354F5&lang=en

**Comment by exocritic5457** [comment]
The 2 stars are offset in their orbits and not completely opposite of each other. Credit where credit is due though, thanks for putting AB before the letters of the circumbinary planets unlike what so many others don't do

**Reply by Karistus** [reply]
> The 2 stars are offset in their orbits and not completely opposite of each other.
Because i forgot to add the AscendingNode value for Exhibit-2 B before uploading, i'll update the addon later.

Appreicate the feedback!

**Reply by Karistus** [reply]
Should be updated now
